src/schema/paddle.ts

```diff
@@ -0,0 +1,100 @@
+import type { CountryCode } from '@paddle/paddle-node-sdk';
+import type { BaseContext } from '../Context';
+import { paddleInstance } from '../common/paddle';
+import { remoteConfig } from '../remoteConfig';
+import type { IResolvers } from '@graphql-tools/utils';
+import { traceResolvers } from './trace';
+
+export const typeDefs = /* GraphQL */ `
+  """
+  Price details for a product
+  """
+  type Price {
+    """
+    Label of the price
+    """
+    label: String!
+    """
+    Id of the price
+    """
+    value: String!
+    """
+    Formatted price with currency symbol
+    """
+    price: String!
+    """
+    Raw unformatted price value
+    """
+    priceUnformatted: String!
+    """
+    Three letter currency code (e.g. USD, EUR)
+    """
+    currencyCode: String!
+    """
+    Optional additional label text
+    """
+    extraLabel: String
+    """
+    Apps id
+    """
+    appsId: String
+  }
+
+  """
+  Price previews
+  """
+  type PricePreviews {
+    """
+    Three letter currency code
+    """
+    currencyCode: String!
+    """
+    Price previews
+    """
+    items: [Price!]!
+  }
+
+  extend type Query {
+    pricePreviews: PricePreviews!
```

Comment:
Let's make it `@auth`, don't think anonymous users will need the price info anytime soon.

Reply:
And add a test for it.